### PR TITLE
lightweight import of o-grid

### PR DESCRIPTION
**Pull request by wheresrhys**
@i-like-robots

### CODE REVIEW

File: grid/main.scss [hunk 1]
```diff
@@ -1,6 +1,104 @@
...
+
+					// Apply width in %
+					@include _oGridFixSafariWrap($colspan);
+					min-width: oGridColspan($colspan);
+					max-width: oGridColspan($colspan);
+					width: oGridColspan($colspan);
+				}
+			}
+		}
+	}
```
Comment (i-like-robots) [inline grid/main.scss]: As discussed elsewhere, I don't like this as it creates so much cruft... I can only find a few uses of it within o- components but I wonder if we could add a warning here saying you should generate a separate stylesheet or something?

Comment (wheresrhys) [inline grid/main.scss]: don't like what exactly?

Comment (i-like-robots) [inline grid/main.scss]: Usage of this mixin. IMO we should where possible encourage generating a separate stylesheet for old IE at a fixed breakpoint.

Comment (wheresrhys) [inline grid/main.scss]: let's revisit before releasing v1.

**Pull request merged**